cms/users/signal_handlers.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Any
+
+from django.contrib.auth.signals import user_logged_in, user_logged_out, user_login_failed
+from django.dispatch import receiver
+from django.http import HttpRequest
+
+from cms.core.utils import get_client_ip
+
+from .models import User
+
+logger = logging.getLogger("cms.users")
+
+
+@receiver(user_logged_in)
+def audit_user_logged_in(sender: Any, request: HttpRequest, user: User, **kwargs: Any) -> None:  # pylint: disable=unused-argument
+    logger.info(
+        "User logged in",
+        extra={
+            "user_id": user.id,
+            "email": user.email,
+            "event": "user_logged_in",
+            "ip_address": get_client_ip(request),
```

Comment:
As per the [dissemination guidance](https://github.com/ONSdigital/dp-standards/blob/main/LOGGING_STANDARDS.md#sensitive-information), they do not want apps to log ip addresses. Where has this requirement come from?

Reply:
I was told explicitly to log IP address - specifically for auth events. I'll raise it again and double check.

Reply:
It's a requirement, especially useful for auditing events. I've opened a PR to the standards so it's more widely visible. https://github.com/ONSdigital/dp-standards/pull/17. 

I think proper conclusion of that PR blocks this one.

Reply:
We’re unlikely to see progress on this any time soon, as any changes here would require a clear and tangible justification due to the significant governance work involved, particularly around DPIA and GDPR compliance.

In the interest of moving this on, I suggest we revert the IP logging addition for now so the merge can proceed. If it's later agreed that IP logging is acceptable, we can revisit and implement it with the necessary approvals.

Reply:
Sounds sensible to me

Reply:
Removed in https://github.com/ONSdigital/dis-wagtail/pull/135/commits/1141a66e614f7c83f71a1401e156a1cde2f146e0. I'll open it as a separate PR once this is merged